src/connectors/sogreatandpowerful.ts

```diff
@@ -0,0 +1,5 @@
+export {};
+
+Connector.artistSelector = 'h1';
+Connector.trackSelector = '#info';
+Connector.playerSelector = '#player';
```

Comment:
This could be replaced with:

```js
Util.bindListeners(
	['audio'],
	['playing', 'pause', 'timeupdate'],
	Connector.onStateChanged,
);
```

Since we can bind to the `audio` element to watch for events from that.